partner_communication_compassion/models/lifecycle_events.py

```diff
@@ -39,7 +39,7 @@ def process_commkit(self, commkit_data):
                         {
                             "config_id": communication_type.id,
                             "partner_id": sponsor.id,
-                            "object_ids": lifecycle.child_id.id,
+                            "object_ids": lifecycle.child_id.sponsorship_ids.ids,
```

Comment:
The problem is that some communications are still connected to the child:
![image](https://user-images.githubusercontent.com/8435180/207069789-30fd0f00-1ebf-4f89-aad2-9e8a1bf46fa2.png)
So this change should only happen if the `communication_type` is a PlannedExit or UnplannedExit. We should also only take the first sponsorship.
```suggestion
                            "object_ids": lifecycle.child_id.sponsorship_ids[:1].ids,
```